src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp

```diff
@@ -534,12 +574,56 @@ void ScalarEmitter::emit_isa(const std::vector<size_t> &in, const std::vector<si
     h->uni_vbroadcastss(vmm_dst, table_val("scalar"));
 }
 
+#ifdef CPU_DEBUG_CAPS
+void ScalarEmitter::print_debug_info() const {
+    std::cerr << "Emitter type name:" << get_type_name(this) << "\n";
+    std::cerr << "where value:" << value << "\n";
+}
+#endif
+
 MemoryEmitter::MemoryEmitter(jit_generator* h, cpu_isa_t isa, const ExpressionPtr& expr) : jit_emitter(h, isa) {
     const auto n = expr->get_node();
     src_prc = n->get_input_element_type(0);
     dst_prc = n->get_output_element_type(0);
 }
 
+#ifdef CPU_DEBUG_CAPS
+void MemoryEmitter::memory_track(size_t gpr_idx_for_mem_address) const {
+    h->push(h->r15);
+    h->push(h->r14);
+    Xbyak::Label label_set_address_current;
+    Xbyak::Label label_set_address_end;
+    h->mov(h->r15, reinterpret_cast<size_t>(&start_address));
+    h->cmp(h->qword[h->r15], 0);
+    h->jne(label_set_address_current);
+    h->mov(h->r14, reinterpret_cast<size_t>(&byte_offset));
+    h->mov(h->r14, h->qword[h->r14]);
+    h->add(h->r14, Xbyak::Reg64(gpr_idx_for_mem_address));
+    h->mov(h->qword[h->r15], h->r14);
+    h->jmp(label_set_address_end);
+    h->L(label_set_address_current);
+    {
+        h->mov(h->r15, reinterpret_cast<size_t>(&current_address));
+        h->mov(h->r14, reinterpret_cast<size_t>(&byte_offset));
+        h->mov(h->r14, h->qword[h->r14]);
+        h->add(h->r14, Xbyak::Reg64(gpr_idx_for_mem_address));
+        h->mov(h->qword[h->r15], h->r14);
+    }
+    h->L(label_set_address_end);
+    // iteration++, 1 means first access
+    h->mov(h->r15, reinterpret_cast<size_t>(&iteration));
+    h->add(h->qword[h->r15], 0x01);
+    h->pop(h->r14);
+    h->pop(h->r15);
+}
+
+void MemoryEmitter::print_debug_info() const {
+    std::cerr << "Emitter type name:" << get_type_name(this) << std::endl;
```

Comment:
Can we use DEBUG_LOG macro?

Reply:
DEBUG_LOG macro is in several lines above, which include 2 functions. But now these changes is removed.